torchmultimodal/utils/attention.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
why do we need a attention.py under utils? just add these to the existing attention module?

Reply:
I envisioned `modules/layers/attention.py` to contain primarily `nn.Modules`/classes intended to be used as layer/components/buildling blocks for models. Utility functions such as `get_extended_attention_mask` seemed to belong more in the `utils/` folders as opposed to `modules/layers/attention.py`.